docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,284 @@
+    - option: "4"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the first part."
+    - option: "2"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the last part."
+```
+
+The rendered exercise type depends largely on the distractors block. In the
+example above, the use of the `distractor_output` field indicates this exercise
+will be a select output exercise.
+
+Other distractor types will result in different exercise types:
+
+- distractor_output -> [Select Output](select-output.md)
+- distractor_code   -> [Select Code](select-code.md)
+- distractor_blanks -> [Tap](tap.md)
+- distractor_text   -> [Multiple Choice](multiple-choice.md)
+- distractor_table  -> [Select Table](select-table.md)
+
+The only exercise type that does not require a distractor field is the
+[Reorder](reorder.md) .
+
+Regardless of the exercise type, the distractor field is always a list of
+distractors (AKA wrong answers). Each individual distractor in the list has two
+elements: `option`, the incorrect answer itself, and `feedback`, the feedback
+message that students will see if they incorrectly choose that distractor.
+
+# Other available blocks
+
+## table
+
+```yaml
+table:
+  data: |-
+    name     ,birthdate
+    50 Cent  ,1975-07-06
+    Aaliyah  ,1979-01-16
+    Aaron Yoo,1979-05-12
+  message: "Showing 3 out of 537 rows"
+```
+
+You can opt to use a `table` block instead of an `output` block, for example
+when displaying the output of a SQL query, which is more appropriate as a
+table. The table field has two subfields. First is `data`, which contains the
+actual table data as comma separated values. They needn't be aligned, but they
+can be and sometimes this improves readability.  The second is `message`, which
+contains a short string of text that is displayed in the footer of the
+table. The most common use for the table message is to indicate row truncation,
+because tables should not be displayed with more than 5 rows as a general rule.
+
+## image
+
+```yaml
+image: my-cool-plot
+```
+
+You can put image assets into your exercises! In this example, `my-cool-plot` is
+a reference to an image asset specified in the [course
+manifest](../repo-structure.md).
+
+## complex context
+
+The `context` block can be more complex than a simple string. Sometimes you need
+to introduce a table, image, multi-line code snippet, or output in the context,
+*before* the question block. This really starts to stretch the limits of what
+can be displayed on a single screen, so simple text context should always be
+strongly preferred.
+
+For example, to include an image such as a plot:
+
+```yaml
+context:
+  text: Study the plot below to answer the following question.
+  image: my-cool-plot
+```
+
+To include a table:
+
+```yaml
+context:
+  text: Study the table below to answer the following question.
+  table: |-
+    foo, bar, baz
+    1  ,2   ,3
+    4  ,5   ,6
+```
+
+## feedback
```

Comment:
Maybe don't mention this? We don't support that right?

Reply:
We do support it, we brought it back